ui/app/AppLayouts/Profile/views/wallet/AccountView.qml

```diff
@@ -36,6 +36,7 @@ Item {
             id: header
             spacing: Style.current.smallPadding
             StatusSmartIdenticon {
+                objectName: "accountImage"
```

Comment:
I think it would be safer to rename those to something more specific like `walletAccountViewAccountImage` just to be sure they don't clash with other names. (some for the ones below)

Reply:
thats true. will do :)